app/Console/Commands/ImportPatreonPledgers.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Patreon;
+use App\Services\Patreon\PatreonApi;
+use App\Services\Patreon\Resources\Pledge;
+use App\Services\Patreon\Resources\Reward;
+use Illuminate\Console\Command;
+
+class ImportPatreonPledgers extends Command
+{
+    protected $signature = 'import:patreon-pledgers';
+
+    protected $description = 'Import pledgers from Patreon';
+
+    /** @var \App\Services\Patreon\PatreonApi */
+    protected $api;
+
+
+    public function __construct(PatreonApi $api)
+    {
+        $this->api = $api;
+
+        parent::__construct();
+    }
+
+    public function handle()
```

Comment:
This method body is somewhat long. Let's make it shorter by moving some logic to a method called `getPledges`